src/editor/components/menu/view-mode-toggle-button.jsx

```diff
@@ -33,14 +41,26 @@ export class ViewModeToggleButtonUnconnected extends React.Component {
         classes={{ tooltip: "iodide-tooltip" }}
         title={this.props.tooltipText}
       >
-        <Button
-          style={this.props.style}
-          onClick={this.toggleViewMode}
-          variant="text"
-          mini
-        >
-          {this.props.buttonText}
-        </Button>
+        <div>
+          {this.props.isReportView && (
+            <Button
+              style={this.props.style}
+              onClick={this.printReport}
+              variant="text"
+              mini
+            >
+              Print
```

Comment:
Could probably use a printer icon as opposed to a textual button here, I'm sure material ui has one.